docs/source/getting_started/primitives.ipynb

```diff
@@ -149,6 +149,22 @@
     "ft.list_primitives().head(5)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For a DataFrame of metrics that summerizes various properties and capabilities of all of the built-in primitives in Featuretools, call `ft.summarize_primatives()`."
```

Comment:
So I didn't realize this until now, but do we want to pivot this DataFrame? It's a bit hard to read currently (you have to scroll all the way to the right)

Reply:
I just uploaded it not too long ago. hmm idk I like it horizontal personally. We can limit the columns shown like we limit the rows shown on `ft.list_primitives`?

Reply:
@tamargrey @thehomebrewnerd @rwedge Any opinions on which looks better?

Reply:
My vote would be for the output of `ft.summarize_primitives()` to be a single column with multiple rows. 

Also noticed some mis-spelling of primitive (`primatives`) in the column headers that needs to be fixed.

Reply:
I think I agree with the single column approach. (also, on a completely separate note--should `list_primitives` be in the api reference? I was gonna suggest adding `summarize_primitives` and notice it wasn't there)

Reply:
Yes it should be api_reference.rst (in `docs/source` folder)

Reply:
Okay, take a look at the most recent commit. lmk if you guys are okay with the naming conventions, doc strings etc...

Reply:
also in the docs I shortened summerize_primitives() to the first 5 rows like list_primitves. doesn't look like the table is as large so I can make this a higher number if desired.